packages/ButtonGroup/ButtonGroupItem/ButtonGroupItem.jsx

```diff
@@ -3,10 +3,69 @@ import PropTypes from 'prop-types'
 
 import { componentWithName, or } from '@tds/util-prop-types'
 
-import safeRest from '../../../shared/utils/safeRest'
+import styled from 'styled-components'
+
+import { borders, forms } from '@tds/shared-styles'
+import { medium, boldFont } from '@tds/shared-typography'
+import { media } from '@tds/core-responsive'
+import { colorTelusPurple, colorWhite } from '@tds/core-colours'
 import generateId from '../../../shared/utils/generateId/generateId'
+import safeRest from '../../../shared/utils/safeRest'
+
+const baseButton = {
+  margin: 0,
+  padding: '0 2rem',
+  cursor: 'pointer',
+  background: 'none',
+  transition: 'background 0.2s',
+  display: 'flex',
+  width: '100%',
+  alignItems: 'center',
+  justifyContent: 'center',
+  ...media.from('md').css({
+    display: 'inline-flex',
+    width: 'auto',
+    minWidth: '136px',
+  }),
+}
 
-import styles from './ButtonGroupItem.modules.scss'
+const StyledButtonGroupItem = styled.div({
+  margin: '0.5rem 0',
+})
+const StyledInput = styled.input({
+  position: 'fixed',
+  opacity: '0',
+  '&:focus ~ label': {
+    boxShadow: `0px 0px 0px 2px ${colorTelusPurple}, 0px 0px 8px 1px ${colorTelusPurple}`,
```

Comment:
This appears to have incorrect styling when a button is focused.

Currently it looks like this:
![image](https://user-images.githubusercontent.com/10531523/56910425-8f0b1a00-6a78-11e9-9feb-2c5e5c08a705.png)

When it should look like this:
![image](https://user-images.githubusercontent.com/10531523/56910212-02f8f280-6a78-11e9-801e-e0209d14ca3f.png)

This may be the result of `boxShadow` being being overridden.